keda/templates/keda-vpa.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if .Values.vpa.keda.enabled }}
+apiVersion: autoscaling.k8s.io/v1
+kind: VerticalPodAutoscaler
+metadata:
```

Comment:
Operator metadata are like this:
```
  name: {{ .Values.operator.name }}
  namespace: {{ .Release.Namespace }}
  annotations:
    {{- toYaml .Values.additionalAnnotations | nindent 4 }}
  labels:
    app: {{ .Values.operator.name }}
    name: {{ .Values.operator.name }}
    app.kubernetes.io/name: {{ .Values.operator.name }}
    {{- include "keda.labels" . | indent 4 }}
```
So I would have that has well here :)

Reply:
i didn't add the annotations because I'm not sure they're needed for the vpa.
I also only included the `keda.labels` as labels.